source/assets/stylesheets/application.scss

```diff
@@ -1,2 +1,2 @@
-@import "./partials/index";
-@import "govuk-frontend/govuk/all.scss";
\ No newline at end of file
+@import "govuk-frontend/govuk/all.scss";
+@import "./partials/index";
```

Comment:
Need to add: 
$govuk-images-path: "../images/";
$govuk-fonts-path: "../fonts/";

Before the govuk library